SyncUp.Web/src/components/AddGroupButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-btn
```

Comment:
Just to note, I rolled my edit and create dialog into one component. Thoughts on doing that here? You'd also want to decouple the button from this component.